schema/src/CreditClassInfo.yaml

```diff
@@ -0,0 +1,90 @@
+id: https://framework.regen.network/schema/CreditClassInfo
+name: CreditClassInfo
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+  qudt: http://qudt.org/schema/qudt/
+  unit: http://qudt.org/vocab/unit/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - taxonomy
+  - SDG
+  - ProjectBenefit
+  - PrimaryImpact
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  CreditClassInfo:
+    class_uri: rfs:CreditClassInfo
+    abstract: true
+    title: Credit Class info
+    description: Base class with common fields used to define credit class info.
```

Comment:
I learned that there are additional [common-metadata](https://linkml.io/linkml-model/latest/docs/CommonMetadata/) that we can use on all parts of the data model. The `title` is nice because we can give a more human-readable string that is rendered on top of markdown pages (`Credit Class info` instead of `CreditClassInfo`)

`abstract` is also useful to demonstrate that this is an abstract base class and should not be instantiated/used directly.